site/specs-source/voice.json

```diff
@@ -1815,7 +1820,7 @@
         "tags": [
           "Conferences"
         ],
-        "summary": "Get Member Information",
+        "summary": "Get Conference Member",
```

Comment:
If you're changing this to Conference Member, I'd change `Update Member` to `Update Conference Member `for consistency.